dev/tests/acceptance/tests/functional/Magento/FunctionalTest/ConfigurableProductCatalogSearch/Test/EndToEndB2CGuestUserTest.xml

```diff
@@ -27,4 +27,23 @@
         <grabAttributeFrom selector="{{StorefrontProductInfoMainSection.productImage}}" userInput="src" stepKey="searchGrabConfigProductPageImageSrc" after="searchAssertConfigProductPage"/>
         <assertNotRegExp expected="'/placeholder\/image\.jpg/'" actual="$searchGrabConfigProductPageImageSrc" stepKey="searchAssertConfigProductPageImageNotDefault" after="searchGrabConfigProductPageImageSrc"/>
     </test>
+    <test name="EndToEndB2CGuestUserMysqlTest">
```

Comment:
Please, revert changes, files in the ConfigurableProductCatalogSearch module are part of Magento core, not MSI project

Reply:
Reverted changes